web/client/configs/localConfig.json

```diff
@@ -413,6 +413,13 @@
     ],
     "desktop": [
       "Details",
+      {
+        "name": "DynamicLegend",
+        "cfg": {
+          "isFloating": false,
+          "flatLegend": false
+        }
+      },
```

Comment:
This plugin should be available only in contexts so it should be removed from localConfig.json